### PR TITLE
add page faq-contact end style about_us

### PR DESCRIPTION
**Pull request by MrSlesh**

### CODE REVIEW

File: public/locales/ru.json [hunk 1]
```diff
@@ -262,7 +262,7 @@
 		"wishlist": "Избранное",
 		"about_us1": "Независимо от того, являетесь ли вы опытным коллекционером, ищущим редкие артефакты, или продавцом, заинтересованным в привлечении глобальной аудитории, Allcom предоставляет платформу, отвечающую вашим потребностям. Наши аукционы включают в себя разнообразные категории, включая искусство, антиквариат, ювелирные изделия, винтажные товары и многое другое.",
 		"about_us2": "Так же наша компания в скором времени запустит возможность доставки, об этом мы сразу сообщим на сайте.",
-		"about_us3": "Локация нашего склада указана на карте для удобства Вашего поиска. Находимся Мы в городе Berlin по адресу Wilhelminenhofstraße 93, 12459."
+		"about_us3": "Локация нашего склада указана на карте для удобства Вашего поиска. Находимся Мы в городе Берлин по адресу Вильхельминенхоф штрассе 93, 12459."
```
Comment (Igor-Klein-1988) [inline public/locales/ru.json:277]: It's better to leave it in German

File: src/assets/scss/elements/_about_us.scss [hunk 2]
```diff
@@ -1,33 +1,46 @@
...
+}
+
+.about-us-container {
+  position: relative;
+  padding: 20px;
+  border-radius: 5px;
+  margin: 20px auto;
+  max-width: 1350px;
+  background-color: #f9f9f9;
+  color: #333;
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_about_us.scss]: use constant

File: src/assets/scss/elements/_about_us.scss [hunk 3]
```diff
@@ -1,33 +1,46 @@
...
+  box-shadow: var(--border-box-shadow);
+}
+
+.about-us-header {
+  font-size: 24px;
+  font-weight: bold;
+  color: #333;
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_about_us.scss]: use constant

File: src/assets/scss/elements/_contact.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+.contact-page {
+  position: relative;
+  padding: 25px;
+  border-radius: 5px;
+  margin: 10px;
+  background-color: #f9f9f9;
+  max-width: 1350px; 
+  color: #333;
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_contact.scss]: use constant

File: src/assets/scss/elements/_faq.scss [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  }
+}
+
+.faq-answer {
+  display: none;
+  padding: 10px;
+}
+
+.faq-question.expanded {
+  background-color: #ccc;
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_faq.scss]: use constant

File: src/features/Contact/Contact.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import '../../assets/scss/elements/_contact.scss';
+import { useTranslation } from 'react-i18next';
+
+const Contact: React.FC = () => {
```
Comment (Igor-Klein-1988) [inline src/features/Contact/Contact.tsx:4]: create `main` folder and move this component there

File: src/features/Contact/Contact.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import '../../assets/scss/elements/_contact.scss';
+import { useTranslation } from 'react-i18next';
```
Comment (Igor-Klein-1988) [inline src/features/Contact/Contact.tsx:2]: move in 2 line. See `readme` file

File: src/features/FAQ/FAQ.tsx [hunk 8]
```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+
+import '../../assets/scss/elements/_faq.scss';
+import { useTranslation } from 'react-i18next';
+
+const FAQPage: React.FC = () => {
```
Comment (Igor-Klein-1988) [inline src/features/FAQ/FAQ.tsx:4]: create main folder and move this component there

File: src/features/Contact/Contact.tsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import '../../assets/scss/elements/_contact.scss';
```
Comment (Igor-Klein-1988) [inline src/features/Contact/Contact.tsx]: Are you sure it doesn't work without it

File: src/features/FAQ/FAQ.tsx [hunk 10]
```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+
+import '../../assets/scss/elements/_faq.scss';
```
Comment (Igor-Klein-1988) [inline src/features/FAQ/FAQ.tsx]: Are you sure it doesn't work without it

**Review by maiorovvv — DISMISSED** [review]

**Review by Igor-Klein-1988 — DISMISSED** [review]

File: public/locales/de.json [hunk 11]
```diff
@@ -279,5 +279,57 @@
 		"storage_info": "Speicherinformationen",
 		"btn": "Produkt hinzufügen",
 		"product_foto": "Produktfoto"
+	},
+	"faq": {
+	"faq":"FAQ",
```
Comment (Igor-Klein-1988) [inline public/locales/de.json]: use formatter

File: src/assets/scss/elements/_about_us.scss [hunk 12]
```diff
@@ -1,33 +1,46 @@
...
-	}
-	&-leaflet-map-container {
-    position: relative;
-    display: flex;
-    align-items: center;
-		justify-content: center;
-		text-align: center;
-    z-index: 1;
-	}
+body {
```
Comment (DenysBabenko29) [inline src/assets/scss/elements/_about_us.scss]: why change styles in Body?

File: src/assets/scss/elements/_contact.scss [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+  margin: 10px;
+  background-color: #f9f9f9;
+  max-width: 1350px; 
+  color: var(--primary-color);
+  box-shadow: var(--border-box-shadow);
+  width: 75%;
+  margin: 20px auto;
+}
+
+.h1-contact {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_contact.scss]: start with contact-

File: src/assets/scss/elements/_contact.scss [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+  display: flex;
+  justify-content: space-between;
+  background-color: #f9f9f9;
+
+  @media screen and (max-width: 1200px) {
+    flex-direction: column;
+  }
+}
+
+.left-content-contact,
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_contact.scss]: same

File: src/assets/scss/elements/_contact.scss [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+.right-content-contact {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_contact.scss]: same

File: src/assets/scss/elements/_contact.scss [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+  width: 45%;
+  margin-top: 20px;
+
+  @media screen and (max-width: 1200px) {
+    width: 100%;
+    margin: 0;
+  }
+}
+
+.left-content-contact {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_contact.scss]: same

File: src/assets/scss/elements/_contact.scss [hunk 17]
```diff
@@ -0,0 +1,101 @@
...
+
+  @media screen and (max-width: 1200px) {
+    margin-right: 0;
+    margin-left: 0;
+    margin-top: 10px;
+    max-width: 100%;
+  }
+}
+
+.left-content-contact p {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_contact.scss]: same

File: src/assets/scss/elements/_contact.scss [hunk 18]
```diff
@@ -0,0 +1,101 @@
...
+  text-align: center; 
+  hyphens: auto;
+  margin-bottom: 25px;
+
+  @media screen and (max-width: 1200px) {
+    margin-bottom: 15px;
+  }
+}
+
+.left-content-contact ul {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_contact.scss]: same

File: src/assets/scss/elements/_faq.scss [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.faq-question.expanded {
+  background-color: #ccc; 
+}
+
+.faq-question.expanded + .faq-answer {
+  display: block;
+}
+.bietschritte-table {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_faq.scss]: same

File: src/assets/scss/elements/_faq.scss [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+  width: 100%;
+  border-collapse: collapse;
+  margin-top: 10px;
+}
+
+.bietschritte-table th, .bietschritte-table td {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_faq.scss]: same

File: src/assets/scss/elements/_faq.scss [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+  border: 1px solid #ddd;
+  padding: 8px;
+  text-align: left;
+}
+
+.bietschritte-table th {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_faq.scss]: same

File: src/assets/scss/elements/_privacy_policy.scss [hunk 22]
```diff
@@ -0,0 +1,66 @@
...
+
+.privacy-policy-question.expanded + .privacy-policy-answer {
+  display: block;
+}
+
+.privacy-policy-answer li {
+  margin-bottom: 10px;
+}
+
+.bietschritte-table {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_privacy_policy.scss]: same

File: src/assets/scss/elements/_privacy_policy.scss [hunk 23]
```diff
@@ -0,0 +1,66 @@
...
+  margin-top: 10px;
+}
+
+.bietschritte-table th, .bietschritte-table td {
+  border: 1px solid #ddd;
+  padding: 8px;
+  text-align: left;
+}
+
+.bietschritte-table th {
```
Comment (Igor-Klein-1988) [inline src/assets/scss/elements/_privacy_policy.scss]: same

**Review by Igor-Klein-1988 — APPROVED** [review]

**Pull request merged**